src/tools/lint-docs/src/lib.rs

```diff
@@ -141,12 +141,11 @@ impl<'a> LintExtractor<'a> {
             let (doc, name) = loop {
                 match lines.next() {
                     Some((lineno, line)) => {
-                        let line = line.trim();
-                        if let Some(text) = line.strip_prefix("/// ") {
-                            doc_lines.push(text.trim().to_string());
```

Comment:
I'm not sure why this was `trim`ing the docs, but this is what was causing the lack of indentation.